### ISSUE TITLE
Make *codesearch-results* buffer read-only

### ISSUE THREAD

**Issue by wyuenho** [abingham/codesearch.el#23]
Currently the results buffer remains writable after the codesearch process has written to it. It should probably be set to read only after so the user can avoid editing the buffer while navigating the results.

**Comment by wyuenho** [comment]
Or better yet, make the results pop up a temp buffer window like *Help* does.

**Comment by abingham** [comment]
Thanks! I've updated `listing-codesearch.el` to make `*codesearch-results*` read only after it is populated. However...

This repository is no longer active. I apologize for not marking it as such, and I'll do so soon. The new repository for this project is [here](https://github.com/abingham/emacs-codesearch). That's where the active development is, including the [commit that fixes this issue](https://github.com/abingham/emacs-codesearch/commit/2419c3dd02cd6c060d8cca07974e7c9b767d62b3). I suggest that you switch to that. Again, sorry for the confusion.

**Comment by abingham** [comment]
> Or better yet, make the results pop up a temp buffer window like Help does.

This is a fine idea, and I'd accept a PR that did this, but I probably won't do it myself. More recent versions of `codesearch.el` include `helm-codesearch.el` which provides very nice helm integration for codesearch...this is very much like what you're looking for, I think.

**Comment by wyuenho** [comment]
Not everyone uses helm, it's just too slow. I can look into popping up a temp buffer window. In the meantime can you switch Melpa's codesearch package to the new one?

**Comment by abingham** [comment]
I've created [a PR with melpa](https://github.com/melpa/melpa/pull/5133/) to update the repo link. Thanks for helping me clean stuff up. I had lost the thread of this project a bit...too much going on!

**Comment by wyuenho** [comment]
Will projectile-codesearch continue to work with the new emacs-codesearch?

**Comment by abingham** [comment]
Actually I just removed that because it had bit-rotted to the point where
it didn't work. But it should be straightforward to make it work again. Let
me add that back in...

**Comment by abingham** [comment]
I've created [a PR with Melpa](https://github.com/melpa/melpa/pull/5135) to use [the new location for `projectile-codesearch.el`](https://github.com/abingham/emacs-codesearch/blob/master/projectile-codesearch.el). Hopefully it'll get sorted soon.

I apologize for the chaotic state of this project. Hopefully it's more sane and sensible now. I don't get much feedback from users on it, and without that I just make sure it works for me. Your help getting this sorted has been really valuable.
